test/integration/nkilib/experimental/moe/test_bwmm_per_expert_scale.py

```
@@ -0,0 +1,674 @@
+    mean_abs = torch.mean(torch.abs(ref.float() - out_per_expert.float())).item()
+    close = torch.allclose(ref.float(), out_per_expert.float(), rtol=5e-2, atol=1e-2)
+
+    print(f"  Max  |ref - kernel|: {max_abs:.6e}")
+    print(f"  Mean |ref - kernel|: {mean_abs:.6e}")
+    print(f"  Allclose(rtol=5e-2): {'PASS ✅' if close else 'FAIL ❌'}")
+
+    if not close:
+        print("  [NOTE] Diff may be due to FP8 quantization error, not a kernel bug.")
+        print("         Check magnitude vs expected FP8 quantization noise.")
+
+    print(f"\n  ✅ Baremetal test completed.")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)